doc/files/package.json.md

```diff
@@ -170,6 +170,16 @@ Both email and url are optional either way.
 
 npm also sets a top-level "maintainers" field with your npm user info.
 
+## support
+
+You can specify a URL for up-to-date information about ways to support
+development of your package:
+
+    { "support": "https://example.com/project/support" }
```

Comment:
Would it be worth versioning this attribute so we can make changes to it in the future? It would allow us to add complexity over time as we see the need.

Reply:
Heh. The latter is what I was thinking.

```
{
  "support": {
    "version": "1",
    "url": "https://example.com/project/support"
  }
}
```

But also it might be too weird to version individual attributes rather than the `package.json` schema as a whole.

Reply:
This does collide with what we are defining in https://github.com/nodejs/package-maintenance/blob/master/docs/drafts/PACKAGE-SUPPORT.md.

It would be good to avoid a conflict with that in that we'll be suggesting package maintainers add the larger structure.

Would what is suggested in maintenance/blob/master/docs/drafts/PACKAGE-SUPPORT.md be possible if qualified through a later version?  And if so how would we have to update what is in there if we wanted to suggest the larger structure start to be used today?  Alternatively, we could change "support" to something else but then it would potentially duplicate info.